### PR TITLE
WIP: Setmar simple ensemble init

### PR DESCRIPTION
**Pull request by havahol**
This is how I think we should initialize the ensemble. As clean and simple as possible.

TODO:
- Make sure that all parameters in a CDKLM sim produced by the DoubleJetCase.IEWPFPaperCase is found in the NetCDF file, and used when initiating a new sim with this file.
- Finish `generate_truth.py`.

I will finish the structure of `generate_truth.py` tomorrow morning (either with or without writing to json), and will commit my final version to this branch/PR.

### CODE REVIEW

**Comment by havahol** [comment]
I managed to get stuck with a nasty bug after you left, but both scripts for generating initial conditions and a truth finally works :) PR ready for review.

File: gpu_ocean/demos/DAPaper/scripts/generate_ensemble_init.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+parser = argparse.ArgumentParser(description='Generate an ensemble.')
+parser.add_argument('-N', '--ensemble_size', type=int, default=10)
+
+args = parser.parse_args()
+
+
+# Checking input args
+if args.ensemble_size > 100:
+    print("Are you really sure that you want to generate more than 100 ensemble members?")
+    print("They will nonethereless look similar to each other, and two identical members " + \
```
Comment (setmar) [inline gpu_ocean/demos/DAPaper/scripts/generate_ensemble_init.py:42]: Typo/fill word. Suggestion: Remove "nonethereless".

File: gpu_ocean/demos/DAPaper/scripts/generate_ensemble_init.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+          "will still diverge due to model errors soon enough.")
+    sys.exit(-1)
+elif args.ensemble_size < 1:
+    parser.error("Illegal ensemble size " + str(args.ensemble_size))
+
+    
+# Define suitable folder for creating the ensemble in
+folder_name = "ensemble_init_" + datetime.datetime.now().strftime("%Y_%m_%d-%H_%M_%S") + "/"
+print("------ Generating initial ensemble ---------------")
+print("Writing ensemble to director: " + folder_name)
```
Comment (setmar) [inline gpu_ocean/demos/DAPaper/scripts/generate_ensemble_init.py:52]: Typo: "directory".

File: gpu_ocean/demos/DAPaper/scripts/generate_truth.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+#--------------------------------------------------------------
+# Initialize the truth from the Case (spin up is done in the Case)
+#--------------------------------------------------------------
+
+
+toc = time.time()
+print("\n{:02.4f} s: ".format(toc-tic) + "Generated initial conditions for truth")
+
+
+#netcdf_args = {}
```
Comment (setmar) [inline gpu_ocean/demos/DAPaper/scripts/generate_truth.py:131]: Probably safe to remove :)

**Review by setmar — APPROVED** [review]
Notebooks run OK. Relevant unit tests OK.

**Pull request merged**